.github/workflows/changelog_update.yml

```
@@ -0,0 +1,29 @@
+          bundler-cache: true
+      - name: Configure Git
+        run: |
+          git config user.name "GitHubActions"
+          git config user.email "<>"
+      - name: Generate and push changelog
+        run: |
+          bundle exec github_changelog_generator -u svck-dev -p ikibana;
+          git add CHANGELOG.md;
+          git commit -m "[skip ci] Update changelog";
+          git push;
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)